packages/plugin-user/src/user.js

```diff
@@ -193,21 +248,49 @@ const User = SparkPlugin.extend({
   },
 
   /**
-   * Determines if the speicfied user needs to signup or can signin
+   * Determines if the speicfied user needs to signup or can signin.
+   * Triggers activation email if client credentials are used
    * @param {Object} options
-   * @param {string} options.email
+   * @param {string} options.email (required)
+   * @param {string} options.reqId required for the endpoint
    * @returns {Promise<Object>}
    */
   verify(options) {
     options = Object.assign({}, this.config.verifyDefaults, options);
 
-    return this.request({
-      service: `atlas`,
-      resource: `users/email/verify`,
-      method: `post`,
-      body: options,
-      requiresClientCredentials: true
-    })
+    if (!options.email) {
+      return Promise.reject(new Error(`\`options.email\` is required`));
+    }
+
+    if (!options.reqId) {
+      return Promise.reject(new Error(`\`options.reqId\` is required`));
+    }
+
+    let shouldRefreshAccessToken = true;
+    let requiresClientCredentials = false;
+    let headers;
+
+    return this.spark.credentials.getAuthorization()
```

Comment:
getAuthorization() should be marked as deprecated. I think you want getUserToken()